ctapipe/image/predictor.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         ----------
         tel_positions : u.Quantity[length]
             Position of the telescopes in the frame of the array
-        tel_pointings : u.Quantity[Angle]
+        tel_pix_coords_altaz : u.Quantity[Angle]
             Pointing of the pixels for each telescope in AltAz
```

Comment:
What is the shape here? `[n_telescopes, n_pixels]`? Should be added to the docstring

Reply:
Ah yes sry. I'll change it